wallet-new/src/Cardano/Wallet/Kernel/DB/Sqlite.hs

```diff
@@ -317,10 +350,12 @@ unsafeMigrateMetaDB conn =
 -- | Simply a conveniency wrapper to avoid 'Kernel.TxMeta' to explicitly
 -- import Sqlite modules.
 newConnection :: FilePath -> IO Sqlite.Connection
-newConnection = Sqlite.open
+newConnection path = Sqlite.open path
+-- TODO: add a debug option for this: Sqlite.setTrace conn (Just putStrLn)
```

Comment:
If you wanted to do that, you can ask for a logging function from the wallet layer on initialization, it gets passed one. That function takes a logging priority, so you could pass a "debugging" priority there.

Reply:
I change the TODO comment. setTrace is actually documented as a dangerous function and should run only in debug mode whatever the logger is. Maybe I will try to pass here the --wallet-debug  option and only then apply it.

Reply:
Ok, sure. Was merely pointing out that the logger is available should you need it :)